canonical-web-and-design/jaas-dashboard
# Upgrade dependencies

## Done

CRA has now released the fix that broke the demo builds. This updates it and the other outdated packages.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The app should work as expected

## Details

Fixes #336
